contracts/zero-ex/contracts/src/features/BundleSwapFeature.sol

```diff
@@ -108,9 +113,14 @@ contract BundleSwapFeature is
     }
 
     function _swap(Swap calldata swap) private returns (Result memory result) {
+        // REVIEW: the called sub-features already do some logic
```

Comment:
I wanted to do it this way. In this case I would need to use delegatecall instead of call. And this leads to a problem for some features when bundling ETH-token and token-token swaps. Therefore I had to do it like this. in this case the sub-features don't try to transfer from sender wallet to itself but from bundleswap to itself. For all my tests this works so far. We also have it like that in 31Third's BatchTrade. 
I first tried the other option with delegatecall because it would have been more efficient but I stumbled across the mentioned issues.

Reply:
thanks for the explanation but this needs more investigation for next steps imo